### PR TITLE
Hecu real not fake

### PR DESCRIPTION
**Pull request by starch70**
## About the PR

adds hecu spawner and job shit
adds jumpsuit and filled backpack for hecu

also adds jobtracker .ftl shit for NTR and BSO because they were just. Missing. Didnt exist. Strange.

## Why / Balance

bored
i need counter to the HEV suit gordon larpers

## Media

image
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: HECU operative job/loadout (admeme, ghostrole, like how ERT is) and a jumpsuit for them

## Summary by CodeRabbit

- **New Features**
	- Introduced the "HECU Operative" role with unique characteristics, job description, and starting gear.
	- Added a new "HECU uniform" and associated gear, including a filled backpack with weapons and medical supplies.
	- Implemented a new PDA variant, "HecuPDA," and HECU ID card for enhanced gameplay experience.
	- Added playtime tracking for the HECU Operative role.
	- Introduced the "Blueshield Officer" role with distinct attributes.

- **Bug Fixes**
	- Corrected indentation for the Helldiver ID card entity.

- **Documentation**
	- Added metadata for clothing sprites to improve asset management.

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
consider the following
1. make them human only (like half-life)
2. use the regular chest rig instead of the merc webbing, it will represent their actual colour scheme better (keep the merc backpack though
3. dont put their magazines in their bag, make a fill of the webbing and put the magazines (and maybe some grenades) in there instead
4. make them use the merc web vest instead of the syndicate web vest
5. the echis pistol is for borgs and recharges on its own, DONT use that
6. there is a way to make them spawn with mindshields automatically implanted
7. give them unique helmets that use the colour scheme correctly
8. give them jackboots instead of combat boots
image
image

**Comment by starch70** [comment]
> consider the following
> 
> 1. make them human only (like half-life)
> 2. use the regular chest rig instead of the merc webbing, it will represent their actual colour scheme better (keep the merc backpack though
> 3. dont put their magazines in their bag, make a fill of the webbing and put the magazines (and maybe some grenades) in there instead
> 4. make them use the merc web vest instead of the syndicate web vest
> 5. the echis pistol is for borgs and recharges on its own, DONT use that
> 6. there is a way to make them spawn with mindshields automatically implanted
> 7. give them unique helmets that use the colour scheme correctly
> 8. give them jackboots instead of combat boots
>    image
>    image

yeah i know the echis is for borgs i dont care and im too lazy to do the shit that makes them mindshielded automatically

**Comment by Piras314** [comment]
real

**Comment by starch70** [comment]
im too lazy to fix the conflicts rn tbh

File: Resources/Prototypes/_Goobstation/Roles/play_time_trackers.yml [hunk 1]
```diff
@@ -3,3 +3,6 @@
 
 - type: playTimeTracker
   id: JobNanotrasenRepresentative
+
+- type: playTimeTracker
+  id: JobHecuOperative
```
Comment (Aidenkrz) [inline Resources/Prototypes/_Goobstation/Roles/play_time_trackers.yml:8]: why

Comment (starch70) [reply Resources/Prototypes/_Goobstation/Roles/play_time_trackers.yml:8]: tbh i kinda just saw it in PRs that added ERT roles and thought it was needed for any new job

**Review by whateverusername0 — APPROVED** [review]

**Pull request merged**